src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/SkipVote.java

```diff
@@ -40,21 +40,23 @@
     private boolean onFailed;
     private boolean onUnstable;
     private boolean onNotBuilt;
+    private boolean onAborted;
 
     /**
      * Standard DataBound Constructor.
-     *
      * @param onSuccessful if the vote should be skipped (not counted) for {@link hudson.model.Result#SUCCESS} builds.
      * @param onFailed if the vote should be skipped (not counted) for {@link hudson.model.Result#FAILURE} builds.
      * @param onUnstable if the vote should be skipped (not counted) for {@link hudson.model.Result#UNSTABLE} builds.
      * @param onNotBuilt if the vote should be skipped (not counted) for {@link hudson.model.Result#NOT_BUILT} builds.
+     * @param onAborted if the vote should be skipped (not counted) for {@link hudson.model.Result#ABORTED} builds.
      */
     @DataBoundConstructor
-    public SkipVote(boolean onSuccessful, boolean onFailed, boolean onUnstable, boolean onNotBuilt) {
+    public SkipVote(boolean onSuccessful, boolean onFailed, boolean onUnstable, boolean onNotBuilt, boolean onAborted) {
```

Comment:
Maybe I'm too _nitty_ here. But to follow the dogma the old constructor signature should be left but deprecated.

Reply:
Hello, 
I implemented backward compatibility and would like review comment. One of the problems is the annotation (@DataBoundContrustor) that can not be retained on the original method.

https://github.com/keesj-exset/gerrit-trigger-plugin/commit/cf03b2148f8e7c85e94cf2fa3f534162288bccb7a

Reply:
No the `@DataboundConstructor` should always move to the correct method that the form post is supposed to call. So it's correct to move that.